templates/manager/email-instance.html

```diff
@@ -31,10 +24,28 @@ <h1 class="h2 heading-underline">{{instance.email.title}} Instance</h1>
       <strong>Note:</strong> This instance was cancelled manually. <span class="fas fa-info-circle" data-toggle="tooltip" data-placement="top" title="This instance was cancelled before emails were sent to all recipients."></span>
     </div>
   {% endif %}
-  <div class="row">
+  <div class="row align-content-center">
     <div class="col-lg-7 col-sm-12 mb-3">
-      <p class="mb-1">Started on {{instance.start}}{% if not instance.in_progress %} Ended on {{instance.end}}{% endif %}</p>
-      <p>This instance was scheduled to go to <strong>{{instance.recipient_details.count}}</strong> recipients. <strong>{{instance.sent_count}}</strong> were actually sent.</p>
+      <dl class="row">
+        <dd class="col-md-4">Instance Subject</dd>
```

Comment:
I know mobile is not that important for Postmaster, so this is not necessary, but if we want this list to collapse better in mobile we could add these classes to all of the `dd` elements:
`mb-0 mb-md-2`